### ISSUE TITLE
Board Editor: Move and rotate unaligns devices

### ISSUE THREAD

**Issue by 5n8ke** [LibrePCB/LibrePCB#728]
##### VERSION / OS / ENVIRONMENT

```
LibrePCB Version: 0.1.4
Git Revision:     ae04eef5a71b5ba66ae2cee6b631c1c933ace535
Build Date:       2020-05-02T18:50:43
Qt Version:       5.11.3 (built against 5.11.3)
CPU Architecture: x86_64
Operating System: Windows 10 (10.0)
Platform Plugin:  windows
```

##### SUMMARY
When a device, aligned to the snap grid, is selected, moved with left hold, and then rotated with right-click, the alignment is destroyed.

##### STEPS TO REPRODUCE
Open a board with a device aligned to the snap grid, select with left click, move with left hold, and rotate 1-3 times with right-click.

##### EXPECTED RESULTS
An aligned device is still aligned to the snap grid.

##### ACTUAL RESULTS
The device is no longer aligned.

##### ADDITIONAL INFORMATION
- It seems, that the rotation origin does not match the displayed device origin.
- The offset rotation origin seems to change with the relation of the actual mouse pointer in regard to the device origin. A bigger distance also results in a bigger offset of the origin.
- After four rotations, the device is again aligned with the snap grid.

**Comment by doom-fr** [comment]
Right. I have reproduced it.
Same with rotation using **'r'** key

**Comment by doom-fr** [comment]
I think this is related : the text moves when rotating

![LibrePcb-record-issue728](https://user-images.githubusercontent.com/7809481/108839739-c21b8780-75d5-11eb-851b-fdaa6cdaf31d.gif)

**Comment by ubruhin** [comment]
The original issue should be fixed with #1082 (at least I could not reproduce it anymore). The problem of the moving items after rotation is not fixed yet, but there's already the issue #450 open for that.
